src/Laravel/Exception/ExceptionLog.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+
+namespace Aboalarm\ProviderServicePhpSdk\Laravel\Exception;
+
+/**
+ * Class ExceptionLog
+ * @package Aboalarm\ProviderServicePhpSdk\Laravel\Exception
+ */
+class ExceptionLog extends \Exception
```

Comment:
würde die Klasse anders nennen:
```suggestion
class WebhookClientException extends \Exception
```